.claude/hooks/format-and-lint.py

```
@@ -0,0 +1,195 @@
+import subprocess
+import re
+from pathlib import Path
+
+def log_message(message, level="INFO"):
+    """Log messages with consistent formatting"""
+    print(f"🔧 [{level}] {message}", file=sys.stderr)
+
+def run_command(cmd, cwd=None, timeout=30):
+    """Run shell command with error handling"""
+    try:
+        result = subprocess.run(
```

Ruff (0.12.2)

`subprocess` call with `shell=True` identified, security issue

(S602)